src/docdb/tree/DocDBDatabaseTreeItemBase.ts

```diff
@@ -68,7 +68,7 @@ export abstract class DocDBDatabaseTreeItemBase extends DocDBTreeItemBase<Collec
     // Create a DB collection
     public async createChild(_node: IAzureNode, showCreatingNode: (label: string) => void): Promise<IAzureTreeItem> {
         const collectionName = await vscode.window.showInputBox({
-            placeHolder: `Enter a name for your ${this.childTypeLabel}`,
+            placeHolder: `Enter an Id for your ${this.childTypeLabel}`,
```

Comment:
Also - please keep this change separate

Reply:
Oups, newbie in GitHub mistake - for the next changes I did one branch for each pull request. Sorry